internal/handlers/handlers.go

```diff
@@ -1 +1,59 @@
 package handlers
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func HandleFile(w http.ResponseWriter, r *http.Request) {
+	http.ServeFile(w, r, "./index.html")
+}
+
+func HandleUpload(w http.ResponseWriter, r *http.Request) {
+
+	r.ParseMultipartForm(32 << 20)
+
+	file, header, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "error with upload", http.StatusBadRequest)
+		return
+	}
+
+	defer file.Close()
+
+	forConv, err := io.ReadAll(file)
+	if err != nil {
+		http.Error(w, "error with reading", http.StatusInternalServerError)
```

Comment:
Это проблема клиента, что не отдал весь файл
```suggestion
		http.Error(w, "error with reading", http.StatusBadRequest)
```